src/redux/reducer.ts

```diff
@@ -1,6 +1,47 @@
-export const reducer = (initialState = {}, action) => {
+import { Reducer } from 'redux'
+import { GET_DEFAULT_IMAGES, SEARCH_BREED, SEARCH_BREED_ERROR, TOGGLE_LIKE } from './actions'
+
+export interface DefaultImages {
+  message: string[]
+  status: string
+}
+
+export interface InitialState {
+  images: DefaultImages | any
+  liked: string[] | any
```

Comment:
Can you think of a way to not use `any` here?

Reply:
Oops, added the any in there for debugging, will remove them